ui/packages/catalog-ui-search/src/main/webapp/component/query-interactions/query-interactions.less

```diff
@@ -1,38 +1,32 @@
 @{customElementNamespace}query-interactions {
   .query-interaction,
-  .interaction-icon,
   .interaction-text {
     line-height: @minimumButtonSize;
     height: @minimumButtonSize;
   }
 
   .query-interaction {
     white-space: nowrap;
-    padding: 0px 10px;
+    padding: 0px @largeSpacing;
     cursor: pointer;
-    overflow: hidden;
-  }
-
-  .interaction-icon {
-    text-align: center;
-    width: @minimumButtonSize;
   }
 
-  .interaction-icon,
   .interaction-text {
     display: inline-block;
     vertical-align: top;
-  }
-
-  .interaction-text {
-    width: ~'calc(100% - @{minimumButtonSize})';
     overflow: hidden;
     text-overflow: ellipsis;
+    overflow: hidden;
```

Comment:
`overflow:hidden` twice?

Reply:
Just to make sure that it's actually hidden. 😀 

Removed